docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -838,6 +838,85 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_getFeeRefundTotalsByRecipient
+
+The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been processed for a specific address.
```

Comment:
Can we make "fee refunds" a link to the page in the docs? https://docs.flashbots.net/flashbots-protect/gas-fee-refunds

Reply:
What do you think of saying "earned by" instead of "processed"? I think this is more precise.